src/main/java/org/fundacionjala/sfdc/pageobjects/PageFactory.java

```diff
@@ -0,0 +1,39 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+import org.fundacionjala.sfdc.pageobjects.Home.HomePage;
+import org.fundacionjala.sfdc.pageobjects.Home.HomePageClassic;
+import org.fundacionjala.sfdc.pageobjects.Home.HomePageLightning;
+import org.fundacionjala.sfdc.pageobjects.components.TopMenu;
+import org.fundacionjala.sfdc.pageobjects.components.TopMenuClassic;
+import org.fundacionjala.sfdc.pageobjects.components.TopMenuLightning;
+import org.fundacionjala.sfdc.pageobjects.config.SalesForceAppEnvsConfig;
+
+import static org.fundacionjala.sfdc.pageobjects.SalesForceEnums.Theme;
+
+public class PageFactory {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)